test/src/main/java/org/apache/accumulo/test/ConditionalWriterIT.java

```diff
@@ -1536,4 +1539,33 @@ public void testNoConditions() throws AccumuloException, AccumuloSecurityExcepti
     }
   }
 
+  @Test
+  public void testCreateConditionalWriterUsesClientProps() throws Exception {
+    // Tests that creating a conditional writer includes the client properties that were set
+    String tableName = getUniqueNames(1)[0];
+    var clientProps = getClientProps();
+    // Set non-default values for all conditional writer props
+    clientProps.setProperty(ClientProperty.CONDITIONAL_WRITER_TIMEOUT_MAX.getKey(), "99");
+    clientProps.setProperty(ClientProperty.CONDITIONAL_WRITER_THREADS_MAX.getKey(), "101");
+    clientProps.setProperty(ClientProperty.CONDITIONAL_WRITER_DURABILITY.getKey(),
+        Durability.NONE.name());
+    try (AccumuloClient client = Accumulo.newClient().from(clientProps).build()) {
+      client.tableOperations().create(tableName);
+
+      try (
+          ConditionalWriterImpl cw1 =
+              (ConditionalWriterImpl) client.createConditionalWriter(tableName);
+          ConditionalWriterImpl cw2 = (ConditionalWriterImpl) client
+              .createConditionalWriter(tableName, new ConditionalWriterConfig())) {
```

Comment:
Could test merging of client config and config set via API by trying to set something in the API like the following.  Then should see that value when inspecting config.

```suggestion
              .createConditionalWriter(tableName, new ConditionalWriterConfig().setMaxWriteThreads(200))) {
```